ledger/common/bitutils/utils.go

```diff
@@ -12,8 +12,18 @@ func Bit(b []byte, idx int) int {
 // SetBit sets the bit at position i in the byte array b
 //
 // The function assumes b has at least i bits. The caller must make sure this condition is met.
-func SetBit(b []byte, i int) {
-	byteIndex := i >> 3
-	i &= 7
-	b[byteIndex] |= 1 << (7 - i)
+func SetBit(b []byte, idx int, value int) {
+	byteIndex := idx >> 3
+	idx &= 7
+	mask := byte(1 << (7 - idx))
+	if value == 0 {
+		b[byteIndex] &= ^mask
+	} else {
+		b[byteIndex] |= mask
+	}
+}
```

Comment:
value as input might not be trival on what it means, It might be better to keep both function SetBit and SetBitWithValue even if the first one uses the second one, the use of input of 1 for value param in the ledger might be not easy to read or understand.

Reply:
also user might pass 2 and it might be cause missusage

Reply:
or add the described behaviour to the function comment

Reply:
👍 refactored code to have dedicated `SetBit` and `ClearBit` methods as well as general `WriteBit` method
https://github.com/onflow/flow-go/pull/1800/commits/3426d41a814a6f658e23363e6281e0702fa02a9a